src/components/MoneyRequestConfirmationList.js

```diff
@@ -262,9 +221,9 @@ class MoneyRequestConfirmationList extends Component {
         const selectedParticipants = this.getSelectedParticipants();
         const shouldShowSettlementButton = this.props.iouType === CONST.IOU.MONEY_REQUEST_TYPE.SEND;
         const shouldDisableButton = selectedParticipants.length === 0;
-        const recipient = this.state.participants[0];
+        const recipient = this.props.moneyRequest.participants[0] || {};
```

Comment:
I'm guessing you added the fallback because for redirection we use `useEffect` which takes affect after render so we may render this page with `undefined` participants[0]. Is that the case? If so, can we deal with the navigation delay in another way?

Reply:
Yes, it could be undefined in case we directly access the page. I don't get what do you mean by navigation delay.

Reply:
Yeah, I meant that we should avoid that. This component is rendered in `src/pages/iou/steps/MoneyRequestConfirmPage.js` in which we redirect if the participants array is empty. So ideally the fallback won't be needed since we would never render that component in the first place.

Yet, due to the fact that the redirection is based on `useEffect` https://github.com/Expensify/App/blob/2860ba13901b6910ab0d76f709e363106f1a61fc/src/pages/iou/steps/MoneyRequestConfirmPage.js#L40-L46

the component will be rendered at the very first render then we will proceed with the redirection. Can we fix that?

Let me know if this part was not clear.

Reply:
I'm afraid there is no other fix than using the fallback. The component will always get the chance to be rendered. The fallback is just the same like we do:
`lodashGet(this.props.moneyRequest, 'participants[0]', {})`